test.js

```diff
@@ -77,7 +77,7 @@ describe("FSUpdater", () => {
     });
 
     it("must not be a symlink", () => {
```

Comment:
Maybe we can feature detect the failure case, by doing ` try/catch` + test scenario, if that fails, we can simply switch to use junction here. If we go down this path, leave a comment describing when/how etc this work-around can be removed.